tests/Spec.hs

```diff
@@ -1 +1 @@
-{-# OPTIONS_GHC -F -pgmF hspec-discover -optF --module-name=Spec #-}
+{-# OPTIONS_GHC -F -pgmF hspec-discover -optF --module-name=Spec -Wno-missing-export-lists #-}
```

Comment:
I don't understand the need for the new option, but that's non-blocking.

Reply:
I updated the default warnings, so `missing-export-lists` is now enabled. Since this module is generated, without an export list, it fails unless we silence the warning here.